src/SDKs/CognitiveServices/data-plane/Language/Microsoft.CognitiveServices.Language.Tests/BaseTests.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.CognitiveServices.Language.TextAnalytics;
+using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+using System;
+using System.Net.Http;
+using System.Runtime.CompilerServices;
+
+namespace Microsoft.CognitiveServices.Language.Tests
```

Comment:
Shouldn't the namespace be Microsoft.CognitiveServices.Language.TextAnalytics.Tests? To better match the namespace of the actual API?